clients/web/src/lib/components/abstract/Chatwoot/Chatwoot.types.ts

```diff
@@ -0,0 +1,80 @@
+// =========================
+// Our custom types
+// =========================
+/**
+ * String labels (i.e. flags, tags) that are set on a user's conversation using `window.$chatwoot.setLabel('')`.
+ * These values do *not* persist across sessions/conversations.
+ */
+export type ChatwootConversationLabel = "org-mle";
```

Comment:
string?

Reply:
Will be a union of the acceptable tags. Chatwoot will not apply a tag if it doesn't exist in the portal.